src/Tribe/JSON_LD/Event.php

```diff
@@ -39,11 +39,13 @@ public static function instance( $name = null ) {
 	 * @return string
 	 */
 	private function get_localized_iso8601_string( $date, $event_tz_string ) {
-		$localTime = new DateTime( $date, new DateTimeZone( 'UTC' ) );
-		$localTime->setTimezone( new DateTimeZone( $event_tz_string ) );
-		$localTime->date = $localTime->format( 'c' );
-
-		return $localTime->date;
+		try {
+			$localTime = new DateTime( $date, new DateTimeZone( 'UTC' ) );
+			$localTime->setTimezone( new DateTimeZone( $event_tz_string ) );
+			return $localTime->format( 'c' );
+		} catch ( Exception $e ) {
```

Comment:
It would be cool to use `Tribe__Exception` here to handle the exception. But, since this appears to be a temporary fix, I am guessing you are more concerned with getting this released than polishing it.

Reply:
That's a good call, I have to admit I was blissfully unaware of `Tribe__Exception` until now.

That said, this is indeed a temp fix (pending resolution of [#90984](https://central.tri.be/issues/90984)) and I'm also not sure that `Tribe__Exception` would be a good fit here (even if _WP_DEBUG := true_ I think there are cases where re-throwing the exception would be unhelpful).